From c86be7bd274925b75c6378129042104ec6bc20b9 Mon Sep 17 00:00:00 2001
From: Michael Scott <mike@foundries.io>
Date: Wed, 13 Oct 2021 14:29:16 -0700
Subject: [PATCH 07/45] portenta-m8: drop HUSH_PARSER

Reference for i.MX8MM-EVK dropped this and we're replicating that here.

Signed-off-by: Michael Scott <mike@foundries.io>
---
 configs/portenta-m8_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configs/portenta-m8_defconfig b/configs/portenta-m8_defconfig
index 69a4e5f918..16524f38ef 100644
--- a/configs/portenta-m8_defconfig
+++ b/configs/portenta-m8_defconfig
@@ -37,7 +37,6 @@ CONFIG_SPL_SEPARATE_BSS=y
 CONFIG_SPL_I2C_SUPPORT=y
 CONFIG_SPL_POWER_SUPPORT=y
 CONFIG_NR_DRAM_BANKS=2
-CONFIG_HUSH_PARSER=y
 CONFIG_SYS_PROMPT="u-boot=> "
 # CONFIG_CMD_EXPORTENV is not set
 CONFIG_CMD_IMPORTENV=y
-- 
2.35.1

